docs/docs/migration/index.md

```diff
@@ -2,3 +2,17 @@
 title: Migration
 sidebar_position: 12
 ---
+
```

Comment:
I'd add some general introduction so that nobody thinks that `@comet/upgrade` handles the whole process, e.g.

```suggestion

This chapter contains instructions for migrating from one COMET version to another.

## @comet/upgrade

```

Reply:
sounds good --> 7d7dacf09e779baf648b73cf06e2e49b0c786220